package.json

```diff
@@ -20,14 +20,18 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^1.1.7",
```

Comment:
Hoping you guys stick to only one icon lib.

Reply:
Can we subsequently add more icon libraries as needed?